<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TagEditor</class>
 <widget class="QWidget" name="TagEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>490</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="StateLabel" name="trackNameLabel">
       <property name="text">
        <string>Track:</string>
       </property>
       <property name="buddy">
        <cstring>trackName</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="trackName">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="StateLabel" name="titleLabel">
       <property name="text">
        <string>Title:</string>
       </property>
       <property name="buddy">
        <cstring>title</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="LineEdit" name="title"/>
     </item>
     <item row="2" column="0">
      <widget class="StateLabel" name="artistLabel">
       <property name="text">
        <string>Artist:</string>
       </property>
       <property name="buddy">
        <cstring>artist</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="CompletionCombo" name="artist"/>
     </item>
     <item row="3" column="0">
      <widget class="StateLabel" name="albumArtistLabel">
       <property name="text">
        <string>Album artist:</string>
       </property>
       <property name="buddy">
        <cstring>albumArtist</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="CompletionCombo" name="albumArtist"/>
     </item>
     <item row="4" column="0">
      <widget class="StateLabel" name="composerLabel">
       <property name="text">
        <string>Composer:</string>
       </property>
       <property name="buddy">
        <cstring>composer</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="CompletionCombo" name="composer"/>
     </item>
     <item row="5" column="0">
      <widget class="StateLabel" name="albumLabel">
       <property name="text">
        <string>Album:</string>
       </property>
       <property name="buddy">
        <cstring>album</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="CompletionCombo" name="album"/>
     </item>
     <item row="6" column="0">
      <widget class="StateLabel" name="trackLabel">
       <property name="text">
        <string>Track number:</string>
       </property>
       <property name="buddy">
        <cstring>track</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="TagSpinBox" name="track"/>
     </item>
     <item row="7" column="0">
      <widget class="StateLabel" name="discLabel">
       <property name="text">
        <string>Disc number:</string>
       </property>
       <property name="buddy">
        <cstring>disc</cstring>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="TagSpinBox" name="disc"/>
     </item>
     <item row="8" column="0">
      <widget class="StateLabel" name="genreLabel">
       <property name="text">
        <string>Genre:</string>
       </property>
       <property name="buddy">
        <cstring>genre</cstring>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="CompletionCombo" name="genre"/>
     </item>
     <item row="9" column="0">
      <widget class="StateLabel" name="yearLabel">
       <property name="text">
        <string>Year:</string>
       </property>
       <property name="buddy">
        <cstring>year</cstring>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="TagSpinBox" name="year"/>
     </item>
     <item row="10" column="0">
      <widget class="StateLabel" name="ratingLabel">
       <property name="text">
        <string>Rating:</string>
       </property>
       <property name="buddy">
        <cstring>ratingWidget</cstring>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <layout class="QHBoxLayout" name="ratingLayout">
       <item>
        <widget class="RatingWidget" name="ratingWidget" native="true"/>
       </item>
       <item>
        <widget class="QLabel" name="ratingVarious">
         <property name="text">
          <string>&lt;i&gt;(Various)&lt;/i&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="11" column="0">
      <widget class="StateLabel" name="commentLabel">
       <property name="text">
        <string>Comment:</string>
       </property>
       <property name="buddy">
        <cstring>comment</cstring>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="LineEdit" name="comment"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="PlainNoteLabel" name="label_7x">
     <property name="text">
      <string>Multiple genres should be separated via a comma (e.g. 'Rock,Hard Rock')</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="PlainNoteLabel" name="ratingNoteLabel">
     <property name="text">
      <string>Ratings are stored in an external database, and &lt;b&gt;not&lt;/b&gt; in the song's file.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progress"/>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>2</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>support/lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>CompletionCombo</class>
   <extends>QComboBox</extends>
   <header>widgets/completioncombo.h</header>
  </customwidget>
  <customwidget>
   <class>TagSpinBox</class>
   <extends>QSpinBox</extends>
   <header>widgets/tagspinbox.h</header>
  </customwidget>
  <customwidget>
   <class>StateLabel</class>
   <extends>QLabel</extends>
   <header>widgets/statelabel.h</header>
  </customwidget>
  <customwidget>
   <class>PlainNoteLabel</class>
   <extends>QLabel</extends>
   <header>widgets/notelabel.h</header>
  </customwidget>
  <customwidget>
   <class>RatingWidget</class>
   <extends>QWidget</extends>
   <header>widgets/ratingwidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>trackName</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
